pkg/api/types.go

```diff
@@ -673,6 +673,9 @@ type GlusterfsVolumeSource struct {
 	// Required: EndpointsName is the endpoint name that details Glusterfs topology
 	EndpointsName string `json:"endpoints"`
 
+	// Optional: Servers is the gluster server list as a string seperated by a comma.
+	Servers string `json:"servers,omitempty"`
```

Comment:
I think it make more sense to use []string regardless how gluster use it. I am updating the PR.. Thanks @erictune for your review!!